Scripts/Lib/epub_generation.py

```diff
@@ -0,0 +1,520 @@
+                )
+
+                if stripped_line:
+                    (
+                        new_line,
+                        new_previous_is_break,
+                        new_previous_is_subpart,
+                        new_previous_is_split,
+                    ) = self.process_line(
+                        stripped_line,
+                        previous_is_break,
+                        previous_is_subpart,
+                        previous_is_first,
+                        previous_is_split,
+                    )
+                    previous_is_break = new_previous_is_break
+                    previous_is_subpart = new_previous_is_subpart
+                    previous_is_first = False
+                    previous_is_split = new_previous_is_split
+                    if previous_is_split:
+                        combined_content.append(current_section)
+                        if new_line:
+                            combined_content.append([f'<p class="tx10">{new_line}</p>'])
+                        current_section = []
+                        previous_is_split = False
+                    elif new_line:
+                        if not current_section:
+                            current_section.append(
+                                new_line.replace('<p class="tx">', '<p class="tx1">')
+                            )
+                        else:
+                            current_section.append(new_line)
+
+        if current_section:
+            combined_content.append(current_section)
+        return combined_content
+
+    def generate_chapter_pages(self, chapter_number):
+        return self.replace_text(
+            "<?xml version='1.0' encoding='utf-8'?>\n"
+            '<html xmlns="http://www.w3.org/1999/xhtml" xmlns:epub="http://www.idpf.org/2007/ops" xml:lang="fr" lang="fr">\n'
+            "<head>\n"
+            "<title>WorldEnd2: What Do You Do at the End of the World? Could We Meet Again Once More?, Vol. {BOOK_VOLUME}</title>\n"
+            '<link rel="stylesheet" href="css/stylesheet.css" type="text/css"/>\n'
+            "</head>\n"
+            "<body>\n"
+            '<div class="galley-rw">\n'
+            '<section id="chapter{CHAPTER_NUMBER:03}" class="body-rw Chapter-rw" epub:type="bodymatter chapter">\n'
+            '<div class="image_full"><img alt="Book Title Page" src="images/Art_chapter{CHAPTER_NUMBER:03}.jpg"/></div>\n'
+            "</section>\n"
+            "</div>\n"
+            "</body>\n"
+            "</html>",
+            start=chapter_number,
+        )
+
+    def generate_nav_xhtml(self):
+        text = self.replace_text(
+            "<?xml version='1.0' encoding='utf-8'?>\n"
+            '<html xmlns="http://www.w3.org/1999/xhtml" xmlns:epub="http://www.idpf.org/2007/ops" xml:lang="fr" lang="fr">\n'
+            "<head>\n"
+            "<title>WorldEnd: What Do You Do at the End of the World? Are You Busy? Will You Save Us?, Vol. {BOOK_VOLUME}</title>\n"
```

Comment:
This should be the SukaMoka title, right?

Reply:
True